plat/rpi/rpi5/scmi/scmi_policy.c

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2024 EPAM Systems
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#include <assert.h>
+#include <stdint.h>
+#include <drivers/scmi-msg.h>
+#include <drivers/scmi.h>
+#include <lib/spinlock.h>
+
+#include "rpi5_scmi_resources.h"
+
+static spinlock_t scmi_lock;
+
+static uint32_t rpi_scmi_device_owner[RPI5_SCMI_DEV_COUNT];
+
+int32_t plat_scmi_device_permission(uint32_t agent_id, uint32_t device_id,
```

Comment:
Why you moved this code from `base.c` ? There is nothing RPI5- specific in this functions.

Reply:
Everything here is RPI5 specific actually. Why? because comparing to the bogus RPI5 platform - the normal HW platforms will do much more things here, like configuring firewalls, security settings and etc. And not just saving an Device owner.